### ISSUE TITLE
Add support for Python apps with Javascript assets

### ISSUE THREAD

**Issue by robdimsdale** [paketo-buildpacks/python#473]
We had a [conversation in slack](https://paketobuildpacks.slack.com/archives/CULAS8ACD/p1653502002129949) where a user was trying to build a python app that also had Javascript assets.

There is no built-in order group to support this, and even with custom order groups (i.e. `pack build --buildpack=...`) it still requires the [buildplan buildpack](https://github.com/paketo-community/build-plan) to satisfy the various provides/requires.

This is a use-case that we should support - similar to how the Ruby buildpack allows for Javascript assets.

**Comment by voor** [comment]
Here's an example: https://github.com/excalidraw/excalidraw is a very popular open source React-based application that can run in the web server buildpack scenario (it's a NPM built frontend that can be served from nginx), even skipping the additional complexity of nginx if you just do this:

```
git clone https://github.com/excalidraw/excalidraw
cd excalidraw
pack build --docker-host=inherit excalidraw:latest
```

**Comment by ryanmoran** [comment]
Hmm. I did the following and it appears to build successfully:
```
pack build excalidraw \
  --builder paketobuildpacks/builder-jammy-base:latest \
  --env BP_WEB_SERVER=nginx \
  --env BP_WEB_SERVER_ROOT=build
```

@voor Is there something I'm missing that would cause this to fail?

**Comment by voor** [comment]
Yes, I think there were cached files in my node_modules folder from the host system that were going into the build environment and causing problems.

**Comment by robdimsdale** [comment]
@voor maybe I'm missing something, but the app you linked to doesn't seem to have any python code - it seems mostly typescript/JS with a bit of HTML.

Do you have a link to a different Python app that wraps this app? If not, maybe we could move this issue to the https://github.com/paketo-buildpacks/web-servers repo?

**Comment by ryanmoran** [comment]
I think moving the issue makes sense. My understanding was that this was a JavaScript web app that had a dependency that required Python to build.

**Comment by voor** [comment]
(Referring to deleted comments) Yes, it was because SASS was being built with Python involved, turns out that is not the case, apologies for adding confusion to this issue.  Deleted those comments, and will mention that Python apps needing JavaScript dependencies is something fairly common.

**Comment by robdimsdale** [comment]
@voor no problem - thanks for clarifying 😄 

This issue is probably best to stay where it is as a placeholder for Python apps with Javascript. I would suggest you open an issue at https://github.com/paketo-buildpacks/web-servers/issues (or https://github.com/paketo-buildpacks/nodejs) but given you've deleted your comments I'm going to assume you've got things working.
